packages/e2e/test/artillery/wallet-restoration/WalletRestoration.ts

```diff
@@ -27,27 +23,22 @@ const mapToGroupedAddress = (addrModel: AddressesModel): GroupedAddress => ({
   type: AddressType.External
 });
 
-export const findAddresses: FunctionHook<WalletVars> = async ({ vars }, ee, done) => {
-  vars.walletsCount = Number(env.VIRTUAL_USERS_COUNT);
-  const db: Pool = new Pool({ connectionString: env.DB_SYNC_CONNECTION_STRING });
+export const getAddresses: FunctionHook<WalletVars> = async ({ vars }, _, done) => {
+  vars.walletLoads = Number(env.VIRTUAL_USERS_COUNT);
 
-  try {
-    logger.info('About to query db for distinct addresses');
-    const result: QueryResult<AddressesModel> = await db.query(findAddressesWithRegisteredStakeKey, [
-      vars.walletsCount
-    ]);
-    logger.info('Found addresses count', result.rowCount);
+  const network = networkNameMap[env.KEY_MANAGEMENT_PARAMS.chainId.networkId];
+  const predefinedAddresses: AddressesModel[] = require(`../../predefined-data/addresses/${network}.json`);
```

Comment:
How about
```suggestion
  const predefinedAddresses: AddressesModel[] = require(`../../test-data/addresses/${network}.json`);
```

Reply:
Actually the initial name was exactly `test-data` but changed it to be more explicit :D even though since it's hardcoded there then it is a predefined lol @mkazlauskas

Reply:
28ceb039981803fb27129eafea78f41868f766e3